<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>322</width>
    <height>402</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>322</width>
    <height>402</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>322</width>
     <height>402</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>322</width>
     <height>402</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>322</width>
      <height>402</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>322</width>
      <height>402</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>322</width>
      <height>402</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="gameStart">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="gameTitleLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>55</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>пятнашки</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="settingBtn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>213</width>
        <height>62</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Настройки</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newGameBtn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>213</width>
        <height>62</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Новая игра</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gamePlay">
     <widget class="QPushButton" name="pauseGameBtn">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>20</y>
        <width>87</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Пауза</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitGameBtn">
      <property name="geometry">
       <rect>
        <x>177</x>
        <y>20</y>
        <width>87</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
     <widget class="QPushButton" name="restartGameBtn">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>20</y>
        <width>87</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Заново</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>321</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gameBoardGridLayout"/>
     </widget>
     <widget class="QLabel" name="timerGame">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>17:14</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gameSetting">
     <widget class="QPushButton" name="goStartScreenBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>назад</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>4x4</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>15x15</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Сложность:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Оформление:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>По умолчанию</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Стиль №1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Стиль №2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>100х100</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="text">
       <string>НАСТРОЙКИ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveSettingBtn">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>350</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
